client/my-sites/plugins/plans/index.tsx

```diff
@@ -110,6 +132,69 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 					flowName={ MARKETPLACE_FLOW }
 					shouldShowPlansFeatureComparison
 					isReskinned
+					customRender={ {
+						topButtons: [
+							<td
+								key="current-action"
+								className="plan-features-comparison__table-item is-top-buttons"
+							>
+								<Button className="button plan-features__actions-button is-current" disabled>
+									{ translate( 'Your current plan' ) }
+								</Button>
+							</td>,
+							<td
+								key="business-action"
+								className="plan-features-comparison__table-item is-top-buttons"
+							>
+								{ plugin ? (
+									<CTAButton
+										plugin={ plugin }
+										hasEligibilityMessages={ [] }
+										disabled={ false }
+										plansPage={ false }
+										desiredPlan={
+											intervalType === 'monthly' ? PLAN_BUSINESS_MONTHLY : PLAN_BUSINESS
+										}
+										buttonText={ translate( 'Upgrade to Business' ) }
+										buttonClassName="button plan-features__actions-button is-business-plan"
+									/>
+								) : (
+									<Button
+										className="button plan-features__actions-button is-business-plan"
+										primary
+										href={ `/checkout/${ selectedSite?.slug }/${ PLAN_BUSINESS }` }
+									>
+										{ translate( 'Upgrade to Business' ) }
+									</Button>
+								) }
+							</td>,
+							<td
+								key="ecommerce-action"
+								className="plan-features-comparison__table-item is-top-buttons"
+							>
+								{ plugin ? (
+									<CTAButton
+										plugin={ plugin }
+										hasEligibilityMessages={ [] }
+										disabled={ false }
+										plansPage={ false }
+										desiredPlan={
+											intervalType === 'monthly' ? PLAN_ECOMMERCE_MONTHLY : PLAN_ECOMMERCE
```

Comment:
Doesn't the plans component deal with this?

Reply:
I see the toggle component modify the `intervalType` variable, but can't see how it would handle `PLAN_ECOMMERCE_MONTHLY` or `PLAN_ECOMMERCE` here. Suggestions?

Reply:
I think that's what it does?